src/ptf/dataplane.py

```diff
@@ -14,27 +14,31 @@
 for filters should include a callback or a counter
 """
 
-import sys
+import logging
 import os
-import socket
-import time
 import select
-import logging
+import socket
 import struct
-from collections import defaultdict
-from collections import deque
-from collections import namedtuple
-from threading import Thread
-from threading import Lock
-from threading import Condition
-from . import ptfutils
-from . import netutils
-from . import mask
+import sys
+import time
+from collections import defaultdict, deque, namedtuple
+from threading import Condition, Lock, Thread
+
 import scapy.packet
 import scapy.utils
-from .pcap_writer import PcapWriter
+import six
 from six import StringIO
 
+#  for "backward compatibility"
+if six.PY2:
+    from . import mask, netutils, ptfutils
+    from .pcap_writer import PcapWriter
+else:
+    import ptf.mask
+    import ptf.netutils
+    import ptf.ptfutils
+    from ptf.pcap_writer import PcapWriter
```

Comment:
my python is a bit rusty, these don't work with Python2?

Reply:
`ptf.something` will import with Python 2 & 3, where `from . import something` will work only with the Py2.